braidpool_spec.md

```diff
@@ -98,6 +142,16 @@ The `Braidpool Metadata` is:
 | `miner IP`      | IP address of this miner                                        |
 | [[`parent`, `timestamp`], ...] | An array of block hashes of parent beads and timestamps when those parents were seen |
 
+[kp]
+
+Again, why p2tr?
+
+Why do we need timestamp in the parent pointers? I don't think we need it. Time
+is relative anyway :) All the parent pointer captures is "happens before"
+relationship.
+
+[/kp]
+
```

Comment:
Because block timestamps are delayed and have very poor resolution. The DAA needs higher resolution timestamps.